airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDatabase.java

```diff
@@ -22,6 +22,7 @@
   private static final Duration NETWORK_TIMEOUT = Duration.ofMinutes(1);
   private static final Duration QUERY_TIMEOUT = Duration.ofHours(3);
   private static final SnowflakeSQLNameTransformer nameTransformer = new SnowflakeSQLNameTransformer();
+
```

Comment:
it looks like nothing really was changed here, could you possibly remove this class from the PR?

Reply:
this was changed automatically by fix the code-style job. I will try to remove it, but not sure if the job on CI would pass